recipes/cache-expiration-options/service-worker.js

```diff
@@ -8,10 +8,10 @@
   global.toolbox.options.debug = true;
 
   // Set up a handler for HTTP GET requests:
-  // - '/(.*)' means any URL pathname will be matched.
+  // - /\.ytimg\.com\// will match any requests going against ytimg.com.
   // - toolbox.cacheFirst let us to use the predefined cache strategy for those
   //   requests.
-  global.toolbox.router.get('/(.*)', global.toolbox.cacheFirst, {
+  global.toolbox.router.get(/\.ytimg\.com\//, global.toolbox.cacheFirst, {
```

Comment:
It would, but... I have control over the HTTP requests being made, and I'm not concerned about that happening.